data/ytt/docs/ytt-latest-toc.yml

```diff
@@ -37,6 +39,8 @@ toc:
         url: /lang-ref-ytt-version
       - page: Template module
         url: /lang-ref-ytt-template
+      - page: Schema module
```

Comment:
Heads-up: _all_ modules are listed in `/lang-ref-ytt` and a subset is here.

Let's add the "Schema module" to that file as well, a la "[Overlay module](https://deploy-preview-108--carvel.netlify.app/ytt/docs/latest/lang-ref-ytt/#overlay-module)"

Reply:
I added it at the end of the list.  :)

Reply:
I probably belongs next to where data values are listed, however, all the modules in that section have excerpts of code there, and the schema section doesn't really have anything to put there.